### ISSUE TITLE
Proto definitions not compiling

### ISSUE THREAD

**Issue by goral09** [informalsystems/ibc-rs#2019]
Hey,

I'm using `ibc` and `ibc-relayer` as dependencies:

```
[dependencies.ibc]
git = "https://github.com/informalsystems/ibc-rs"
features = ["std", "clock"]

[dependencies.ibc-relayer]
git = "https://github.com/informalsystems/ibc-rs"
```

but during check phase `cargo check` I'm getting the following errors:
```
error[E0433]: failed to resolve: could not find `query_client` in `v1`
    --> /home/mateusz/.cargo/git/checkouts/ibc-rs-a065dcf743eaa6c9/29877ed/relayer/src/chain/cosmos.rs:1701:52
     |
1701 |                 ibc_proto::ibc::core::channel::v1::query_client::QueryClient::connect(
     |                                                    ^^^^^^^^^^^^ could not find `query_client` in `v1`

error[E0433]: failed to resolve: could not find `query_client` in `v1`
    --> /home/mateusz/.cargo/git/checkouts/ibc-rs-a065dcf743eaa6c9/29877ed/relayer/src/chain/cosmos.rs:1734:52
     |
1734 |                 ibc_proto::ibc::core::channel::v1::query_client::QueryClient::connect(
     |                                                    ^^^^^^^^^^^^ could not find `query_client` in `v1`

error[E0433]: failed to resolve: could not find `query_client` in `v1`
    --> /home/mateusz/.cargo/git/checkouts/ibc-rs-a065dcf743eaa6c9/29877ed/relayer/src/chain/cosmos.rs:1760:52
     |
1760 |                 ibc_proto::ibc::core::channel::v1::query_client::QueryClient::connect(
     |                                                    ^^^^^^^^^^^^ could not find `query_client` in `v1`
```
and some others...

It looks like it can't find the Rust definitions that were supposed to be compiled from the protobuf files. What am I missing? I can't see any `build.rs` that builds them but I also fail to find it in `ibc-relayer-cli` that uses these crates as dependencies as well.

Any help is very welcome, thank you!

**Comment by romac** [comment]
Thanks for the report, we're looking into it. In the meantime, I was able to make a dummy project compile with the following `[dependencies]`:

```toml
[dependencies]

[dependencies.ibc]
git = "https://github.com/informalsystems/ibc-rs"
features = ["std", "clock"]

[dependencies.ibc-relayer]
git = "https://github.com/informalsystems/ibc-rs"

[dependencies.ibc-proto]
git = "https://github.com/informalsystems/ibc-rs"
features = ["tonic"]
```

**Comment by goral09** [comment]
Thank you @romac ! I did not try adding the `ibc-proto` definition explicitly as my expectation was that it will be built as dependency of the `ibc-relayer` itself.

**Comment by romac** [comment]
And you would be right! But there's a typo in [`ibc-relayer/Cargo.toml`](https://github.com/informalsystems/ibc-rs/blob/29877ed44cefc367a81a8c7261907aac76849ca2/relayer/Cargo.toml#L25) which specifies the `tonic` feature whereas it should have been `client`. We'll fix this in 0.13.0 which goes out later today :)

**Comment by romac** [comment]
Fixed in https://github.com/informalsystems/ibc-rs/commit/07b97f9adae2c4fcbfc2c19d987be6d50a661976 which was released in v0.13.0.
